### ISSUE TITLE
replaceAll is not supported

### ISSUE THREAD

**Issue by rmi22186** [yowainwright/es-check#157]
## Requested Update
When we run `es-check` against a file that includes `replaceAll`, `es-check` says the file is valid es5.

## Why Is This Update Needed?
`replaceAll` is an es2021 feature, and so should be flagged when included in an es5 file.

## Are There Examples Of This Requested Update Elsewhere?
No.

**Comment by yowainwright** [comment]
@rmi22186 I _think_ this may be because `replaceAll` all is not a reserved word so there would be no way to identify that `<someString>.replaceAll(<this>, <nowThat>)` wouldn't exist in es5.

I am not against potentially adding the ability to `.escheckrc` to so one could warn/fail if a potential name clash function came about.

```js
{
  es5: ['replaceAll', ...etc]
}
```
